src/sentry/static/sentry/app/views/onboarding/configure/index.jsx

```diff
@@ -7,27 +7,70 @@ import ProjectContext from '../../projects/projectContext';
 import ProjectDocsContext from '../../projectInstall/docsContext';
 import ProjectInstallPlatform from '../../projectInstall/platform';
 
+import Raven from 'raven-js';
+
 const Configure = React.createClass({
   mixins: [ApiMixin],
 
+  getInitialState() {
+    return {
+      isFirstTimePolling: true,
+      hasSentRealEvent: false
+    };
+  },
+
   componentWillMount() {
     let {platform} = this.props.params;
     //redirect if platform is not known.
     if (!platform || platform === 'other') {
       this.redirectToNeutralDocs();
     }
-  },
 
-  componentDidMount() {
+    this.fetchEventData();
     this.timer = setInterval(() => {
       this.fetchEventData();
     }, 2000);
   },
 
+  componentWillUpdate(nextProps, nextState) {
+    if (
+      !this.state.isFirstTimePolling &&
+      nextState.hasSentRealEvent == true &&
+      this.state.hasSentRealEvent == false
+    ) {
+      this.redirectUrl();
+    }
+  },
+
   componentWillUnmount() {
     clearInterval(this.timer);
   },
 
+  sentRealEvent(data) {
+    if (data.length == 1) {
+      let firstError = data[0];
+      return !firstError.message.includes('This is an example');
+    } else {
+      return data.length > 1;
+    }
+  },
+
+  checkPollEventData(data) {
+    let {isFirstTimePolling} = this.state;
+
+    if (isFirstTimePolling) {
```

Comment:
I think rather than checking if `isFirstTimePolling` is true, you could just always say 
```
this.setState({
  isFirstTimePolling: false,
  hasSentRealEvent: this.sentRealEvent(data)
});
```
since there's no condition that would make `isFirstTimePolling` go back to true or change what you set `hasSentRealEvent` to, right?